packages/core-flows/src/inventory/workflows/create-inventory-items.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  WorkflowData,
+  createWorkflow,
+  transform,
+} from "@medusajs/workflows-sdk"
+import {
+  attachInventoryItemToVariants,
+  createInventoryItemsStep,
+  prepareCreateInventoryItems,
+} from "../steps"
+
+import { CreateInventoryItemInput } from "@medusajs/types"
+
+interface WorkflowInput {
+  items: (CreateInventoryItemInput & { _associationTag?: string })[]
```

Comment:
question: I like the idea of the `_associationTag`, but doesn't the `attachInventoryItemToVariants` step defeat its purpose?

Reply:
I'm not sure I understand what you mean.

the `attach` method requires knowing which variant to attach each inventoryItem to. I guess keeping it abstract would be nice if you wanna override the step and attach to something different, but we can also just rename `_associationTag` to `variant_id`

Reply:
Okay got it, so the idea is that with the `associationTag` you only need to override a single step to attach them to something else? 

I am fine to keep the `_associationTag`, but I just find it a bit off to expose this in the workflow input. To me, the `_` prefix signals, that this is a property handled internally. Not a strong opinion though, so if you guys think we should keep it, I am fine with that.

Reply:
I've renamed it to be `tag` from end-to-end just to not rename it halfway through